source/MaterialXGenArnold/ArnoldShaderGenerator.cpp

```diff
@@ -19,6 +20,9 @@ ArnoldShaderGenerator::ArnoldShaderGenerator()
                                       "volume_matte" };
 
     _syntax->registerReservedWords(reservedWords);
+
+    // <!-- <dielectric_brdf> -->
+    registerImplementation("IM_dielectric_brdf_" + ArnoldShaderGenerator::TARGET, ThinFilmSupport::create);
```

Comment:
An unrelated fix, thin-film support was not registered properly for the Arnold generator.